src/System.IO.FileSystem.AccessControl/src/System/IO/FileSystemAclExtensions.net46.cs

```diff
@@ -8,17 +8,17 @@ namespace System.IO
 {
     public static class FileSystemAclExtensions
     {
-        public static DirectorySecurity GetAccessControl(DirectoryInfo directoryInfo)
+        public static DirectorySecurity GetAccessControl(this DirectoryInfo directoryInfo)
```

Comment:
I don't know what the contract is for this function, but I am just fixing the omitted "this" on these extension methods.